techniques/general/G183.html

```diff
@@ -29,16 +31,13 @@ <h3>Colors that would provide 3:1 contrast with black words and 4.5:1 contrast w
       <section class="procedure"><h3>Procedure</h3>
          <ol>
             <li>Locate all instances where color alone is used to convey information about text.</li>
```

Comment:
This should probably be in a paragraph before the list: 
<p>For each instance where color is used to convey information about text:</p>

This will also affect the "checks #2 and #3 are true"